apps/src/templates/manageStudents/ManageStudentsActionsCell.jsx

```diff
@@ -4,6 +4,8 @@ import PopUpMenu, {MenuBreak} from "@cdo/apps/lib/ui/PopUpMenu";
 import color from "../../util/color";
 import FontAwesome from '../FontAwesome';
 import Button from '../Button';
+import {startEditingStudent, cancelEditingStudent} from './manageStudentsRedux';
+import { connect } from 'react-redux';
```

Comment:
nit: would be good to be consistent on whether or not we have styles before/after braces in a single file